src/server/getters/get-reporting-fees.ts

```diff
@@ -306,87 +279,125 @@ function getStakedRepResults(db: Knex, reporter: Address, universe: Address|null
-      if (err) return callback(err);
-      getReporterFeeTokens(db, reporter, universe, feeWindow, (err, totalReporterTokensByFeeWindow: { [feeWindow: string]: BigNumber }) => {
-        if (err) return callback(err);
+    getParticipantEthFees(db, augur, reporter, universe, feeWindow, (err, participantEthFees?: Array<ParticipantEthFee>) => {
+      if (err || participantEthFees == null) return callback(err);
+      getParticipationTokenEthFees(db, augur, reporter, universe, feeWindow, (err, participationTokenEthFees: Array<ParticipationTokenEthFee>) => {
+        if (err || participationTokenEthFees == null) return callback(err);
         getStakedRepResults(db, reporter, universe, feeWindow, (err, repStakeResults) => {
           if (err || repStakeResults == null) return callback(err);
           getMarketsReportingParticipants(db, reporter, currentUniverse, (err, result: FormattedMarketInfo) => {
             if (err || repStakeResults == null) return callback(err);
-            const unclaimedEth = _.reduce(_.keys(totalReporterTokensByFeeWindow), (acc, feeWindow) => {
-              if (totalReporterTokensByFeeWindow[feeWindow].isEqualTo(ZERO)) return acc;
-              const thisFeeWindowTokens = (totalFeeWindowTokens && totalFeeWindowTokens[feeWindow]) || { totalTokens: ZERO, cashBalance: ZERO };
-              const feesForThisWindow = totalReporterTokensByFeeWindow[feeWindow].dividedBy(thisFeeWindowTokens.totalTokens).times(thisFeeWindowTokens.cashBalance);
-              acc = acc.plus(feesForThisWindow);
-              return acc;
-            }, ZERO);
-            const redeemableFeeWindows = _.uniq(_.keys(totalReporterTokensByFeeWindow).concat(repStakeResults.redeemableFeeWindows));
+            const unclaimedParticipantEthFees = _.reduce(participantEthFees, (acc, cur) => acc.plus(cur.ethFees), ZERO);
+            const unclaimedParticipationTokenEthFees = _.reduce(participationTokenEthFees, (acc, cur) => acc.plus(cur.ethFees), ZERO);
+            const redeemableFeeWindows = _.map(participationTokenEthFees, "feeWindow");
             const response = {
               total: {
-                unclaimedEth: unclaimedEth.toFixed(),
-                unclaimedRepStaked: repStakeResults.fees.unclaimedRepStaked.toFixed(),
-                unclaimedRepEarned: repStakeResults.fees.unclaimedRepEarned.toFixed(),
-                lostRep: repStakeResults.fees.lostRep.toFixed(),
+                unclaimedEth: unclaimedParticipantEthFees.plus(unclaimedParticipationTokenEthFees).toFixed(0, BigNumber.ROUND_DOWN),
+                unclaimedRepStaked: repStakeResults.fees.unclaimedRepStaked.toFixed(0, BigNumber.ROUND_DOWN),
+                unclaimedRepEarned: repStakeResults.fees.unclaimedRepEarned.toFixed(0, BigNumber.ROUND_DOWN),
+                lostRep: repStakeResults.fees.lostRep.toFixed(0, BigNumber.ROUND_DOWN),
```

Comment:
For the UI, we need to differentiate between the unclaimed ETH/REP related to the forked market, and all other reporting fees (since there are 2 separate buttons for this). Therefore, I think we'll also need `unclaimedForkRepStaked` and `unclaimedForkEthFees` properties to be returned.